examples/asr/speech_to_text_infer.py

```diff
@@ -49,7 +49,10 @@ def main():
     parser.add_argument("--batch_size", type=int, default=4)
     parser.add_argument("--wer_tolerance", type=float, default=1.0, help="used by test")
     parser.add_argument(
-        "--normalize_text", default=True, type=bool, help="Normalize transcripts or not. Set to False for non-English."
+        "--dont_normalize_text", default=False, action='store_true', help="Turn off trasnscript normalization. Recommended for non-English."
```

Comment:
Why is this arg being inverted ? It's only used for en, and every other language doesn't have normalization ?

Reply:
That's right.  Previous default was True and it is the same behavior. Although previously to turn it off, we would have to pass '--normalize_text ""' to pass an empty string as bool False, which is annoying.

Reply:
Annoying yes, but we don't need to invert the flag to support CER. It will silently create bugs in measuring wer if you forget to pass this flag so it should be reverted.